test/cto/aliasedImport.json

```diff
@@ -16,39 +16,9 @@
                             "$class": "concerto.metamodel@1.0.0.StringProperty",
                             "name": "name",
                             "isArray": false,
-                            "isOptional": false,
-                            "location": {
```

Comment:
Can you generate this file with --excludeLineLocations to make it clearer what is changing, please?

Reply:
The original model does not exist in the repo. So had to do this manually.